podcast/2.19.md

```diff
@@ -0,0 +1,745 @@
+---
+title: 'Creating one CI to rule them all, with Fedor Korotkov, founder and CTO of Cirrus Labs'
+publishDate: 2022-03-15T10:00-07:00
+tags: [podcast]
+slug: fedor-korotkov
+published: true
+---
+
+<!-- START AUDIO -->
+
+<audio className="object-center" src="https://www.buzzsprout.com/1097978/10038427-building-the-code-editor-dreams-are-made-of-with-max-brunsfeld-co-founder-of-zed.mp3?download=true" controls preload="none"></audio>
+
+<!-- END AUDIO -->
+
+<!-- START GUESTS -->
+<span>
+Fedor Korotkov, Beyang Liu
+</span>
+<!-- END GUESTS -->
+
+<!-- START SUMMARY -->
+
+Why can’t one CI scale alongside a company–from startup to enterprise? In this episode, Fedor Korotkov, founder and CTO of Cirrus Labs, joins Beyang Liu, co-founder and CTO of Sourcegraph, to talk about how, as a student back in 2009, he developed a photo app that earned him almost $2,000 a month, share the time he applied to be an intern at Twitter but ended up with a full-time job, and explain how six months of “funemployment” led to the building and founding of Cirrus CI–the one CI to rule them all. Along the way, Fedor explains how Cirrus CI, with Kubernetes, can spin up a new container in two seconds.
```

Comment:
@nickmyyz  @csbailey5t same comment here. The name should be "CirrusLabs," one word. I think the other mentions are all just "Cirrus" in the transcript, so the rest should be okay.